server/src/lib/server.ts

```diff
@@ -71,8 +74,48 @@ app.get('/ping', (_, res) => res.send('pong'));
 
 app.post('/api/validate', corsMiddleware,(request: any, response: any) => {
   const { type, data } = request.body;
-  const result = validator.validator(type, typesPath, JSON.parse(data));
+  let result: Object = {}
+  try {
+    result = validator.validator(type, typesPath, JSON.parse(data));
+  } catch (error) {
+    result = 'Please provide a single valid JSON input'
+  }
   response.json({
    result
   });
 });
+
+app.post('/api/validate-file', corsMiddleware, (request: any, response: any) => {
+  let result: any[] = []
+
+  const busboy = new Busboy({ headers: request.headers, limits: { fields: 1, files: 1 } });
+  let type :any = undefined;
+  busboy.on('field', (fieldname, value, _0, _1, _2, _3) => {
```

Comment:
What is _0 etc used for ?

Reply:
removed unneccessary parameters. please check